pinot-query-runtime/src/test/java/org/apache/pinot/query/runtime/queries/QueryRunnerTest.java

```diff
@@ -258,7 +258,7 @@ private Object[][] provideTestSqlAndRowCount() {
             + "col1, COUNT(*) FROM a GROUP BY col1", 2},
         new Object[]{"SET numGroupsLimit = 2; SELECT /*+ aggOptions(num_groups_limit='1') */ "
             + "a.col2, b.col2, COUNT(*) FROM a JOIN b USING (col1) GROUP BY a.col2, b.col2", 1},
-        new Object[]{"SELECT * FROM \"tbl-escape.naming\"", 5}
+        new Object[]{"SELECT * FROM \"default.tbl-escape-naming\"", 5}
```

Comment:
Even double quoted names are interpreted as database + table name? I would have expected to support things like `select * from db1."some.name.with.dots"`

Reply:
dot is not allowed in table name. I guess both works "dbName"."tbName" and "dbName.tbName"